Tests/js/views/services/assets/ez-contenteditviewservice-tests.js

```diff
@@ -501,4 +844,4 @@ YUI.add('ez-contenteditviewservice-tests', function (Y) {
     Y.Test.Runner.add(eventTest);
     Y.Test.Runner.add(redirectionUrlTest);
     Y.Test.Runner.add(getViewParametersTest);
-}, '', {requires: ['test', 'ez-contenteditviewservice']});
+}, '', {requires: ['test', 'ez-contenteditviewservice', 'array-extras']});
```

Comment:
as far as I can see, there's no call to anything provided by array-extras so that's not needed (since you added it to the dependencies of ez-contenteditviewservice).